src/services/RoutingService.js

```diff
@@ -96,6 +160,27 @@ import { bvvRouteStatsProvider } from './provider/routeStats.provider';
  * @returns {module:domain/routing~RouteStats} the statistics of the route
  */
 
+/**
+ * A function that provides a ETA calculation for a defined categoryId (vehicle type).
+ * @function
+ * @name module:services/RoutingService~RoutingService#getETAFor
+ * @param {string} categoryId
+ * @param {Number} distance distance in meter
+ * @param {Number} elevationUp elevation upwards in meter
+ * @param {Number} elevationDown  elevation downwards in meter
+ * @returns {number|null} the ETA in milliseconds or null if no ETA for the given category is available.
+ */
```

Comment:
Let's move this interface method declaration to the other declarations above
Order should be

1.  RoutingService methods
2. RoutingService provider